packages/ckeditor5-ui/src/editorui/boxed/boxededitoruiview.js

```diff
@@ -107,7 +115,11 @@ export default class BoxedEditorUIView extends EditorUIView {
 		const t = this.t;
 		const voiceLabel = new LabelView();
 
-		voiceLabel.text = t( 'Rich Text Editor' );
+		if ( this.label && this.label !== '' ) {
```

Comment:
If `editorName` was not in `options` the `#label` is `''`. If it was passed in `options`, let's assume it was not `''` and we should use its value as it was passed. This will simplify the code below. This will always come down to `voiceLabel.text = t( 'Rich Text Editor, ' + this.label );`